docs/resources/lakehouse_monitor.md

```diff
@@ -38,15 +37,13 @@ resource "databricks_table" "myTestTable" {
     column {
         name      = "timestamp"
         position  = 1
-        type_name = "INT"
-        type_text = "int"
-        type_json = "{\"name\":\"id\",\"type\":\"integer\",\"nullable\":true,\"metadata\":{}}"
+        type = "int"
     }
 }
 
 resource "databricks_lakehouse_monitor" "testTimeseriesMonitor" {
-    table_name = "${databricks_catalog.sandbox.name}.${databricks_schema.things.name}.${databricks_table.myTestTable.name}"
-    assets_dir = "/Shared/provider-test/databricks_lakehouse_monitoring/${databricks_table.myTestTable.name}"
+    table_name = "${databricks_catalog.sandbox.name}.${databricks_schema.things.name}.${databricks_sql_table.myTestTable.name}"
```

Comment:
This could be simplified to `databricks_sql_table.myTestTable.id` now: https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/sql_table#id